crates/ruff_linter/src/rules/eradicate/detection.rs

```diff
@@ -39,6 +39,15 @@ pub(crate) fn comment_contains_code(line: &str, task_tags: &[String]) -> bool {
         return false;
     };
 
+    // Ignore task tag comments (e.g., "# TODO(tom): Refactor").
+    if line
+        .split(&[' ', ':', '('])
+        .next()
+        .is_some_and(|first| task_tags.iter().any(|tag| tag == first))
```

Comment:
I ended up removing the uppercase check here because it requires an allocation (would be expensive give how often this runs), and is probably rare.

Reply:
Makes sense, thanks!